### DISCUSSION TITLE
deploying code using foundry

### DISCUSSION THREAD

**Discussion by Legacy770** [Cyfrin/foundry-full-course-cu#2780] [Q&A] [answered]
'''
Error: error sending request for url (http://localhost:8545/)
'''

this is the error I am getting while trying to add the anvil rpc url so I can deploy on foundry. I would appreciate any assistance

**Comment by EngrPips** [comment, marked as answer]
Hello @Legacy770, Do you have `Anvil` chain running in a separate terminal?
